src/overlays/actors/ovl_En_Ot/z_en_ot.c

```diff
@@ -41,128 +81,1099 @@ const ActorInit En_Ot_InitVars = {
+    func_8013E1C8(&this->skelAnime, sAnimations, 1, &this->animIdx);
+    this->actor.flags |= 0x8000000;
+    this->actor.flags &= ~(0x8 | 0x1);
+    Actor_SetSwitchFlag(globalCtx, ENOT_GET_3F80(&this->actor));
+    this->actionFunc = func_80B5D750;
+}
+
+void func_80B5D750(EnOt* this, GlobalContext* globalCtx) {
+    if (!(this->unk_32C & 1) && !(this->unk_32C & 2)) {
+        func_8013DE04(globalCtx, &this->unk_2C0, func_8013DF3C, func_80B5D470, func_80B5D37C, func_8013E0A4);
+    }
+
+    Math_Vec3f_Copy(&this->actor.world.pos, &this->unk_330);
+
+    if (this->unk_2C0.unk_1C & 0x40) {
+        this->unk_32C |= 2;
+    }
+
+    if (this->unk_2C0.unk_1C & 0x80) {
+        this->unk_32C |= 1;
+    }
+
+    if ((this->unk_32C & 2) && (this->actor.xyzDistToPlayerSq <= 14400.0f)) {
+        this->unk_32C &= ~2;
+    }
+
+    if ((this->unk_32C & 1) && (this->actor.xzDistToPlayer <= 180.0f)) {
+        this->actor.flags &= ~0x8000000;
+        this->actor.flags |= (0x8 | 0x1);
+        if (D_80B5E884 != 0) {
+            func_80B5C9A8(this, globalCtx);
+        } else {
+            func_80B5CBA0(this, globalCtx);
+        }
+    }
+}
+
+void EnOt_Update(Actor* thisx, GlobalContext* globalCtx) {
+    EnOt* this = THIS;
+    s32 pad;
+
+    if ((this->animIdx == 1) && func_801378B8(&this->skelAnime, this->skelAnime.animFrameCount)) {
+        Audio_PlayActorSound2(&this->actor, NA_SE_EV_SEAHORSE_SWIM);
+    }
+
+    this->actionFunc(this, globalCtx);
+    if (this->actor.bgCheckFlags & 0x20) {
+        if (DECR(this->unk_354) == 0) {
+            if (this->actor.flags & 0x40) {
+                s32 i;
+
+                for (i = 0; i < 2; i++) {
+                    EffectSsBubble_Spawn(globalCtx, &this->actor.world.pos, 0.0f, 20.0f, 5.0f, 0.1f);
+                }
+                this->unk_354 = (Rand_ZeroOne() * 10.0f) + 10.0f;
+            }
+        }
+    }
+
+    Actor_UpdateBgCheckInfo(globalCtx, &this->actor, 0.0f, 10.0f, 0.0f, 4);
+
+    if (this->actor.world.pos.y <= this->actor.floorHeight + 50.0f) {
+        this->actor.world.pos.y = this->actor.floorHeight + 50.0f;
+        this->actor.prevPos.y = this->actor.floorHeight + 50.0f;
+    }
+
+    Actor_SetHeight(&this->actor, 12.0f);
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    Collider_UpdateCylinder(&this->actor, &this->collider);
+    CollisionCheck_SetOC(globalCtx, &globalCtx->colChkCtx, &this->collider.base);
+    func_80B5BB38(&this->unk_747, &this->unk_744, 0.7f);
+
+    if (this->unk_32C & 0x400) {
+        func_80B5BAAC(&this->lightInfo, &this->unk_378, &this->unk_747, 0xD2);
+    }
+
+    func_80B5E078(globalCtx, this->unk_3A4, 10);
+}
+
+void func_80B5DAEC(Actor* thisx, GlobalContext* globalCtx) {
+    EnOt* this = THIS;
+    s32 pad;
+
+    this->actionFunc(this, globalCtx);
+    Actor_SetHeight(&this->actor, 12.0f);
+    SkelAnime_FrameUpdateMatrix(&this->skelAnime);
+    func_80B5BB38(&this->unk_747, &this->unk_744, 0.699999988079f);
```

Comment:
```suggestion
    func_80B5BB38(&this->unk_747, &this->unk_744, 0.7f);
```
I believe. There may be a few others I didn't spot.